compass-rest/src/main/java/de/dfki/asr/compass/rest/SceneNodeRESTService.java

```diff
@@ -62,9 +63,45 @@ public SceneNode getNodeByID(
 		return sceneTreeManager.findById(entityId);
 	}
 
+	@GET
+	@Path("/{parentId}/children")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@ApiOperation(
+			value = "Retrieve a scene node's children.",
+			notes = "Create a new scene node, as a child of the specified parent"
+	)
+	@ApiResponses({
+		@ApiResponse(code = 404, message = "Entity not found")
+	})
+	public List<SceneNode> getChildrenForParent(
```

Comment:
this isn't actually needed, I'll remove it. It only occurred to me while I was writing the PR.